tests/rptest/services/tls.py

```diff
@@ -588,6 +603,11 @@ def create_cert(self,
             f"-in {csr} -out {crt} -extensions server_ext -days {self.cert_expiry_days} -batch"
         )
 
-        cert = Certificate(cfg, key, crt, self.ca)
+        p12_password = self.generate_password(length=16)
+        self._exec(
+            f"openssl pkcs12 -export inkey {key} -in {crt} -certfile {self.ca.crt} "
```

Comment:
Is it worth extracting/reusing some of the commonality in this and the base? I'm guess the argument should be `-inkey` here, like in the base class?

Reply:
sure done